app/controllers/application_controller.rb

```diff
@@ -2,6 +2,10 @@ class ApplicationController < ActionController::Base
   include Slimmer::Headers
   include Slimmer::Template
 
+  before_action do
+    I18n.locale = I18n.default_locale
```

Comment:
Not to block this going out but as a follow-up we should avoid setting I18n.locale directly and globally and instead switch to it being scoped for an action: https://github.com/alphagov/collections/pull/2132/files#diff-766c34fd6533171eaf54300c153f89d6002c35c02cfc9c5b219251f85180ad07R41-R44